test/blackbox-tests/test-cases/include-qualified/basic.t/run.t

```diff
@@ -1,8 +1,27 @@
 Basic test showcasing the feature. Every directory creates a new level of aliasing.
   $ dune build
-  File "lib/dune", line 1, characters 17-26:
-  1 | (include_subdirs qualified)
-                       ^^^^^^^^^
-  Error: Unknown value qualified
-  Hint: did you mean unqualified?
-  [1]
+  hello from nested module bar
```

Comment:
I'm getting an error locally, but I assume this is the output we expect whenever this is done?
```
+  File "exe/test.ml", line 2, characters 2-20:
+  2 |   Foolib.Foo.Bar.run ();
+        ^^^^^^^^^^^^^^^^^^
+  Error: Unbound module Foolib
+  [1]
```

Reply:
Should be fixed now.